boards/px4/fmu-v6xrt/src/board_config.h

```diff
@@ -451,10 +451,12 @@
 #define HRT_PPM_CHANNEL         /* GPIO_EMC_B1_09 GPIO_GPT5_CAPTURE1_1 */  1  /* use capture/compare channel 1 */
 #define GPIO_PPM_IN             /* GPIO_EMC_B1_09 GPT1_CAPTURE2 */ (GPIO_GPT5_CAPTURE1_1 | GENERAL_INPUT_IOMUX)
 
-#define RC_SERIAL_PORT                  "/dev/ttyS4"
-#define RC_SERIAL_SINGLEWIRE            1 // Suport Single wire wiring
-#define RC_SERIAL_SWAP_RXTX             1 // Set Swap (but not supported in HW) to use Single wire
-#define RC_SERIAL_SWAP_USING_SINGLEWIRE 1 // Set to use Single wire swap as HW does not support swap
+#define RC_SERIAL_PORT                     "/dev/ttyS4"
+#define RC_SERIAL_SINGLEWIRE               1 // Suport Single wire wiring
+#define RC_SERIAL_SWAP_RXTX                1 // Set Swap (but not supported in HW) to use Single wire
+#define RC_SERIAL_SWAP_USING_SINGLEWIRE    1 // Set to use Single wire swap as HW does not support swap
+#define RC_SERIAL_SINGLEWIRE_MANUAL_DUPLEX 1 // Set to use ioctl's to manually swap duplex
```

Comment:
Needed?

Reply:
Indentation change, unless you want to keep it as is.

Reply:
But is it used anywhere?

Reply:
https://github.com/PX4/PX4-Autopilot/commit/93a256741ff672a8e863d9fc62c523c0b989b2c2

Reply:
RC_SERIAL_SINGLEWIRE_MANUAL_DUPLEX  ?

Reply:
Good point that's unused and a leftover from the 1st version